README.md

```diff
@@ -77,6 +77,7 @@ All the data (past and coming) are available publicly in JSON:
 * 16: [DDD Melbourne](https://www.dddmelbourne.com/) - Melbourne (Australia)
 * 16: [Global AI Bootcamp 2024 - Mumbai](https://lu.ma/gaib24mum) - Mumbai (India) <a href="https://sessionize.com/gaib24mum/"><img alt="CFP Global AI Bootcamp 2024 - Mumbai" src="https://img.shields.io/static/v1?label=CFP&message=until%2012-March-2024&color=green"></a>
 * 17-18: [Cloud Native Rejekts EU 2024](https://cloud-native.rejekts.io/) - Paris (France)
+* 18-19: [DevOpsDdays Kraków 2024](https://devopsdays.org/events/2024-krakow/welcome/) - Kraków (Poland)
```

Comment:
```suggestion
* 18-19: [DevopsDays Kraków 2024](https://devopsdays.org/events/2024-krakow/welcome/) - Kraków (Poland)
```

Reply:
`DevOpsDays` is the correct spelling.

Source: I am co-chair of the DevOpsDays conference series. :)

Reply:
Nice to meet you :)
Yes I know the spelling but I saw a double d ^^